recipes/scotch/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "scotch" %}
+{% set version = "6.0.4" %}
+{% set md5 = "d58b825eb95e1db77efe8c6ff42d329f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+  url: http://gforge.inria.fr/frs/download.php/file/34618/{{ name }}-{{ version }}.tar.gz
```

Comment:
Could try the link below. The number on it doesn't seem to change with each release. So it works better with templating.

```
https://gforge.inria.fr/frs/download.php/latestfile/298/{{ name }}_{{ version }}.tar.gz
```